corehq/apps/sms/models.py

```diff
@@ -7,6 +7,8 @@
 from django.db import models
 from corehq.apps.users.models import CouchUser, CommCareUser
 from casexml.apps.case.models import CommCareCase
+from dimagi.utils.couch.migration import (SyncCouchToSQLMixin,
```

Comment:
Wow, me neither.  Looks like @gcapalbo just added it: https://github.com/dimagi/dimagi-utils/pull/272